vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -699,4 +699,13 @@
       </ClCompile>
     </ItemGroup>
   </Target>
+  <Target Name="SanityCheck" BeforeTargets="_PrepareForBuild">
```

Comment:
Consider moving this to the external targets included by the template instead of the Microsoft.ReactNative vcxproj. Having the target here means it doesn't affect binary distribution projects.

Reply:
It will happen on customers that build us as source, but in the nuget case this is not actually run...

---
In reply to: [510752706](https://github.com/microsoft/react-native-windows/pull/6310#discussion_r510752706) [](ancestors = 510752706)